MovieFestival/Movie.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MovieFestival
+{
+	internal class Movie
```

Comment:
Why is it internal class?

Reply:
Class Accessibility Level changed to public